### PR TITLE
fix the kindling-collector-config.yml caused by "stat ~/.kube/config: no such file or directory"

### PR DESCRIPTION
**Pull request by yaofighting**
Signed-off-by: yaofighting <siyao@zju.edu.cn>

fix the collector config and make it running normally on the Linux environment
## Description

in the file /root/kindling/collector/docker/kindling-collector-config.yml, we can see a bug about that we may not boot the collector normally in the Linux environment due to the "kube_config_dir" is "~" instead of "root", so in this PR, I fix it.
## Related Issue

https://github.com/Kindling-project/kindling/issues/224
## Motivation and Context

it fix the compatibility of collector
## How Has This Been Tested?

just compile and build the collector container with the doc's step.

### CODE REVIEW

File: collector/docker/kindling-collector-config.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ analyzers:
 processors:
   k8smetadataprocessor:
     kube_auth_type: kubeConfig
-    kube_config_dir: ~/.kube/config
+    kube_config_dir: root/.kube/config
```
Comment (dxsup) [inline collector/docker/kindling-collector-config.yml:76]: Should be started with "/".

**Review by dxsup — CHANGES_REQUESTED** [review]
There is another configuration file under the directory `kindling/deploy`. Although it is using `serviceaccount` as its `kube_auth_type` and `kube_config_dir` isn't effective, I think we better make this field same to avoid misunderstanding.

File: collector/docker/kindling-collector-config.yml (hunk 1 above)
Comment (yaofighting) [reply collector/docker/kindling-collector-config.yml]: OK, it has been modified as required

**Review by dxsup — APPROVED** [review]
LGTM

**Pull request merged**